app/scripts/geocoder/geocoder-service.js

```diff
@@ -18,7 +18,7 @@
         var searchOutFields = 'StAddr,City,Postal';
         var searchCategories = [
             'Street Address',
```

Comment:
Probably want to generalize these two into the 'Address' super-category, and add the 'Postal' category, too. Adding 'Postal' fixed a number of missing street addresses for CAC.